lib/screens/explore_screens/vote_screens/delegate/delegates_tab/delegates_tab.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/components/remove_delegate_confirmation_dialog.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegate_bloc.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegate_event.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegate_page_commands.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegate_state.dart';
+
+class DelegatesTab extends StatelessWidget {
+  const DelegatesTab({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => DelegateBloc()..add(const LoadDelegateData()),
+      child: BlocConsumer<DelegateBloc, DelegateState>(
+        listener: (context, state) {
+          final pageCommand = state.pageCommand;
+
+          if (pageCommand is ShowDelegateRemovalConfirmation) {
+            showDialog<void>(
+              context: context,
+              barrierDismissible: false,
+              builder: (_) {
+                return BlocProvider.value(
+                  value: BlocProvider.of<DelegateBloc>(context),
+                  child: const RemoveDelegateConfirmationDialog(),
+                );
+              },
+            );
+          } else if (pageCommand is ShowOnboardingDelegate) {
+            showDialog<void>(
+              context: context,
+              barrierDismissible: false,
+              builder: (_) => const IntroducingDelegatesDialog(),
+            );
+          } else if (pageCommand is ShowErrorMessage) {
+            SnackBarInfo(pageCommand.message, ScaffoldMessenger.of(context)).show();
+          }
+
+          BlocProvider.of<DelegateBloc>(context).add(ClearPageCommand());
+        },
+        builder: (context, DelegateState state) {
+          switch (state.pageState) {
+            case PageState.initial:
+              return const SizedBox.shrink();
+            case PageState.loading:
+              return const FullPageLoadingIndicator();
+            case PageState.failure:
+              return const FullPageErrorIndicator();
+            case PageState.success:
+              return Padding(
+                padding: const EdgeInsets.symmetric(horizontal: horizontalEdgePadding),
+                child: Column(
+                  children: <Widget>[
+                    const SizedBox(height: 30),
+                    Padding(
+                      padding: const EdgeInsets.symmetric(horizontal: 10),
+                      child: Text(
+                          'Delegating your vote means to entrust the power of your vote to another Citizen.  Please choose your delegate carefully!',
```

Comment:
Did you add ES and BR traslaciones for this text?

Reply:
No translations on this pr